app/views/downloads/_download_preface.html.erb

```diff
@@ -66,10 +66,13 @@
       <ul>
         <% related_works.each do |work| %>
           <li>
-            <% relation_text = work.translation ?
-              "A translation of %{work_link} by %{author_link}" :
-              "Inspired by %{work_link} by %{author_link}"
-            %>
+            <% if work.parent.unrevealed? then
```

Comment:
This won't quite work. The downloads currently show either:

* `A translation of %{work_link} by %{author_link}`
* `Inspired by %{work_link} by %{author_link}`

What we're trying to do is -- for unrevealed works -- replace _just_ the `%{work_link} by %{author_link}` portion with `a work in an unrevealed collection`. 

But what you have will write `a work in an unrevealed collection` without the words `A translation of` or `Inspired by` in front of it.

(Also, minor code style note: we typically don't use `then`.)

So basically we'd need to divide it up more like: 

```
work_info = if work.parent.unrevealed?
              "a work in an unrevealed collection"
            else
              "%{work_link} by %{author_link}"
            end

relation_text = if work.translation
                  "A translation of %{work_info}."
                else
                  "Inspired by %{work_info}."
                end
```